packages/ui/src/components/filters/manage-views.tsx

```diff
@@ -0,0 +1,225 @@
+import { ChangeEvent, FC, memo, useCallback, useEffect, useMemo, useState } from 'react'
+
+import {
+  AlertDialog,
+  AlertDialogContent,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  Button,
+  Icon,
+  Input
+} from '@/components'
+import { DndContext } from '@dnd-kit/core'
+import { SortableContext, useSortable, verticalListSortingStrategy } from '@dnd-kit/sortable'
```

Comment:
when did we add `dnd-kit` as a dependency? 
we need to evaluate all new libraries and get app-sec approvals before we start using them.

Reply:
also, cc @srdjan-harness because we are using some other dnd library for the pipeline studio

Reply:
At the moment, I’ve left comments in the code to revisit this in a future PR.